tools/cldr-apps/js/src/index.js

```diff
@@ -6,14 +6,14 @@ import { createApp } from "vue";
 import AboutPanel from "./views/AboutPanel.vue";
 import WaitingPanel from "./views/WaitingPanel.vue";
 
-function show(component, el) {
-  const app = createApp(component);
-  app.mount(el);
-  return app;
-}
+import * as cldrGui from "../../src/main/webapp/js/esm/cldrGui.js";
```

Comment:
Clever.. I'm ok with this for now though it results in duplicate JavaScript

Reply:
not sure what you mean by duplicate js

Reply:
I meant that `cldrGui.js` would be pulled into the webpack bundle in `bundle.js`, but would also be available separately (unbundled)  under `src/main/webapp/js/esm/cldrGui.js`

Reply:
Yes, all the esm files are available separately, but when they're used in the bundle, they're not used outside the bundle -- in that sense there's no duplication

the browser network tab confirms that

Reply:
@btangmu let's come back to this